operator/internal/component/podclique/pod/syncflow.go

```diff
@@ -232,6 +253,69 @@ func (r _resource) checkAndRemovePodSchedulingGates(sc *syncContext, logger logr
 	return skippedScheduleGatedPods, nil
 }
 
+// isIndividualPodGang determines if a PodGang is an individual PodGang (beyond MinAvailable).
+// Individual PodGangs have names like: "simple1-0-sga-0", "simple1-0-sga-1", etc.
```

Comment:
the comment should have the template format for the podgang name as well. Then an example will make more sense. No one can currently infer what does `sga` stand for or what does it represent.